src/js/_enqueues/lib/nav-menu.js

```diff
@@ -1063,12 +1086,35 @@
 		},
 
 		attachMenuSaveSubmitListeners : function() {
+			const updateNavMenuEl = $( '#update-nav-menu' );
```

Comment:
Same var instead const, Remove empty line.

Reply:
- Resolved in https://github.com/WordPress/wordpress-develop/pull/6389/commits/24598ed85737201ee50112246c4c0b6da3458029 🚀 .